### PR TITLE
Openapi/chore

### PR DESCRIPTION
**Pull request by PierreR**

### CODE REVIEW

File: dhall-openapi/hie.yaml
Comment (sjakobi) [inline dhall-openapi/hie.yaml:1]: I think this is the first `hie.yaml` in this project.

What's the purpose of this file? Which tools can use it?

**Review by sjakobi — COMMENTED** [review]
Why did you add the `dhall-openapi/sandbox/k8s-swagger.json` file?

Regarding the bounds, I suspect that some might be more restrictive than necessary, but this probably doesn't matter.

File: dhall-openapi/hie.yaml
Comment (PierreR) [reply dhall-openapi/hie.yaml:1]: hie.yaml is a file that is used by https://github.com/mpickering/hie-bios (required for `ghcide` or the new integration point: https://github.com/haskell/haskell-language-server)
I can remove it but it won't harm.

Sorry for the `sandbox` folder. I should have checked PR further. Will remove that folder.

Comment (PierreR) [reply dhall-openapi/hie.yaml]: I have squashed both commits, remove both sandbox and `hie.yaml`

File: dhall-openapi/dhall-openapi.cabal
Comment (sjakobi) [inline dhall-openapi/dhall-openapi.cabal:3]: What's the reason for this bump?

**Review by sjakobi — APPROVED** [review]
LGTM!

Adding the `hie.yaml` would have been fine IMHO if it's useful for you or other contributors, @PierreR.

Since I'm not yet familiar with `dhall-openapi`, I'd prefer if @Gabriel439 would review this too before the merge.

File: dhall-openapi/dhall-openapi.cabal
Comment (PierreR) [reply dhall-openapi/dhall-openapi.cabal:3]: @sjakobi I first had 0.1.1 (to be sure I was above the previous fourth digit schema), then 0.1.2 on my second change (remove Turtle). Should I make it  `0.1.1` instead ?

Comment (PierreR) [reply dhall-openapi/dhall-openapi.cabal:3]: Also it would be nice to have someone replacing the current "maintainer" in the cabal file. Maybe @TristanCacqueray could jump in ;-)

Comment (sjakobi) [reply dhall-openapi/dhall-openapi.cabal:3]: Thanks for the clarification! In general, Gabriel takes care of bumping the version numbers during the release process.

Since `dhall-openapi` hasn't been properly released at all so far, no bump seems necessary for now.

Comment (Gabriel439) [reply dhall-openapi/dhall-openapi.cabal:3]: Yeah, there's no need to bump the version until we release.  Also, the version number here is legacy from the `dhall-kubernetes` project.  When we do release the project it will be at version 1.0.0.  I typically don't do `0.*` releases of packages.

**Review by Gabriel439 — APPROVED** [review]

File: dhall-openapi/dhall-openapi.cabal
Comment (TristanCacqueray) [reply dhall-openapi/dhall-openapi.cabal:3]: I am not familiar with the maintainer duties and I think it is better if @Gabriel439 is the selected one for `dhall-openapi`.

**Pull request merged**
